examples/datasource-http/src/DataSource.ts

```diff
@@ -93,7 +102,7 @@ export class DataSource extends DataSourceApi<MyQuery, MyDataSourceOptions> {
       if (_.isString(err)) {
         message = err;
       } else if (isFetchError(err)) {
-        message += err.statusText ? err.statusText : defaultErrorMessage;
+        message = `Fetch error: ${err.statusText ? err.statusText : defaultErrorMessage}`;
```

Comment:
Nit: re-write this to make it more readable, one might think this ternary condition will always return true if you don't look carefully:


```
message = "Fetch error: " + err.statusText ? err.statusText : defaultErrorMessage;
```